packages/terra-menu/src/_MenuUtils.js

```diff
@@ -0,0 +1,57 @@
+const getPopupHeight = (contentHeight) => {
+  if (contentHeight <= 40) {
```

Comment:
We can get rid of this if statement with the Math.ceil and Math.max methods. Perhaps we can switch over to using those?